addons/web_editor/models/ir_ui_view.py

```diff
@@ -106,7 +106,7 @@ def _copy_custom_snippet_translations(self, record, html_field):
                 self._copy_field_terms_translations(custom_snippet_view, 'arch_db', record, html_field)
 
     @api.model
-    def _copy_field_terms_translations(self, record_from, name_field_from, record_to, name_field_to):
+    def _copy_field_terms_translations(self, recordset_from, name_field_from, record_to, name_field_to):
```

Comment:
It's not really stable compliant, this will break custo calling this method with named params (override or not).
But:
- It got introduced very recently in order to improve the custom snippet translation, after years of this behavior not working, hopefully people didn't jump into writing code calling it
- Even if they did, hopefully they did not go with the named param way

AFAIC, this is exceptionally ok given the recent introduction and niche use case of this method.